source/_patterns/02-organisms/content-headers/content-header-profile.yaml

```diff
@@ -0,0 +1,54 @@
+assets:
+  css:
+    - content-header.css
+  js: []
+schema:
+  $schema: http://json-schema.org/draft-04/schema#
+  type: object
+  properties:
+    displayName:
+      type: string
+      minLength: 1
+    details:
+      type: object
+      properties:
+        affiliations:
+          type: array
+          items:
+            type: string
+            minLength: 1
+          minItems: 1
+        emailAddress:
+          type: string
```

Comment:
`type: email` (`minLength` can then be removed).

Reply:
Changing to `type: email` and removing `minLength` gives the validation error:
```
[properties.details.properties.emailAddress.type] Does not have a value in the enumeration ["array","boolean","integer","null","number","object","string"]
[properties.details.properties.emailAddress.type] String value found, but an array is required
```

Reply:
Oh, sorry, it's `type: string, format: email`!

Reply:
That's got it, thanks.